File: Durwapatil/C++ Programs/max-of-two-nos.cpp
// Date : 24 Jan 2021

// CPP program to display the largest number of two numbers



#include <iostream>
using namespace std;

class ab
{
    private : int a, b, c;
    public : int large();
};
int ab :: large()
{
    cout << "Enter two numbers : \n";
    cin >> a >> b;
    if(a > b)
    {
        cout << "Largest number : " << a;
    }
    else
    {
        cout <<  "Largest number : " << b;
    }
}
int main()
{
    ab st;
    st.large();
}


/*
Output : 
    Enter two numbers : 
    12
    49
    
    Largest number : 49
*/

File: Durwapatil/Java/max-of-two-nos.java
// Date : 26 Jan 2021

// Java program to find the maximum of two numbers..



            import java.io.*;
            class max
            {
              public static void main(String args[])
              {
                try
                {
                  DataInputStream x = new DataInputStream(System.in);
                  int a, b;
                  System.out.print("Enter two numbers : \n");
                  a = Integer.parseInt(x.readLine());
                  b = Integer.parseInt(x.readLine());
                  if(a > b)
                  {
                    System.out.print("\n a is maximum");
                  }
                  else
                  {
                    System.out.print("\n b is maximum");
                  }
                }
                catch(Exception e)
                {}
              }
            }


/*
Output : 
    Enter two numbers : 
    5
    9
    
    b is maximum
*/
